scripts/boot_patch.sh

```diff
@@ -185,8 +185,8 @@ rm -f ramdisk.cpio.orig config magisk*.xz
 for dt in dtb kernel_dtb extra; do
   if [ -f $dt ]; then
     if ! ./magiskboot dtb $dt test; then
-      ui_print "! Boot image was patched by old (unsupported) Magisk"
-      abort "! Please restore back to stock boot image"
+      ui_print "! Boot image $dt was patched by old (unsupported) Magisk"
+      abort "! Please restore back to stock boot image and try again"
```

Comment:
```suggestion
      abort "! Please try again with *stock* boot image"
```

Maybe better?

Reply:
To be honest stock might confuse people on custom ROMs. How about "unpatched" ?

Reply:
Sounds good!

```suggestion
      abort "! Please try again with *unpatched* boot image"
```